collect/cache/cache_test.go

```diff
@@ -53,3 +54,43 @@ func TestBufferOverrun(t *testing.T) {
 	c.Set(traces[2])
 	assert.Equal(t, 1, s.CounterIncrements["collect_cache_buffer_overrun"], "buffer should have overrun")
 }
+
+func TestTakeExpiredTraces(t *testing.T) {
+	s := &metrics.MockMetrics{}
+	s.Start()
+	c := &DefaultInMemCache{
+		Config:  CacheConfig{10},
+		Metrics: s,
+		Logger:  &logger.NullLogger{},
+	}
+	c.Start()
+
+	now := time.Now()
+	traces := []*types.Trace{
+		&types.Trace{TraceID: "1", SendBy: now.Add(-time.Minute), Sent: true},
```

Comment:
`Sent` is not used right? Is this just to show that?

Reply:
Right.